prisma/migrations/20240209162415_add_prompt_config/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "prompts" ADD COLUMN     "config" JSONB NOT NULL DEFAULT '{}';
```

Comment:
please update the timestamp of the migration folder so that this is the latest one